docs/tutorial/new_controller.rst

```diff
@@ -0,0 +1,106 @@
+
+Implementing a Controller
+-------------------------
+
+A controller closes the loop, a class derived from the abstract class
+:py:class:`~pymoskito.simulation_modules.Controller`.
+Its only function calculates the controller output,
```

Comment:
Its task is to stabilize the pendulum by calculating a suitable input for the model.